examples/CRISP/scripts/setup.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# This is all stuff that has to happen after the source code is mounted 
+# TOOD: perhaps we can try and move more of this to the dockerfile build process
+# Eg. copy package.json and Cargo.toml and then try to build out dependencies however this is relatively complex
+(cd /app && git submodule update)
```

Comment:
submodules aren't initialized
Use `git submodule update --init --recursive` instead

Reply:
done - still getting the same error.

Reply:
okay found the issue - we need to add `platform: linux/amd64` on `docker-compose` since ARM64 architecture (aarch64) is not supported on Apple Silicon Mac, so we need to explicitly tell Docker to build the image using amd64 architecture (even if I'm on ARM).

Reply:
Does `platform: linux/amd64` in `docker-compose` works for you?

Reply:
If not then we might need to do a manual install on aarch64 
https://dev.risczero.com/api/zkvm/install#manual-installation-and-installation-for-all-other-systems-eg-x86-64-macos-arm64-linux

using `ARG TARGETARCH & TARGETPLATFORM`

Reply:
> Does `platform: linux/amd64` in `docker-compose` works for you?

yes, solves the issue.